### PR TITLE
ODE update

### PR DESCRIPTION
**Pull request by lacava**
adds equations for strogatz problems to the metadata

### CODE REVIEW

File: datasets/strogatz_shearflow1/metadata.yaml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 dataset: strogatz_shearflow1
 description: |
     This is one state of a 2-state dynamic model for shear flow. 
+    
+    $\dot{\theta} = \cot (\phi) \cdot cos(\theta)$
+    $ \dot{\phi} = \left(\cos ^2 (\phi) + 0.1 \cdot  \sin^2 (\phi)\right) \cdot sin(\theta)$
```
Comment (trangdata) [inline datasets/strogatz_shearflow1/metadata.yaml]:
```suggestion
    $\dot{\theta} = \cot (\phi) \cdot \cos(\theta)$
    $ \dot{\phi} = \left(\cos ^2 (\phi) + 0.1 \cdot  \sin^2 (\phi)\right) \cdot \sin(\theta)$
```

File: datasets/strogatz_glider2/metadata.yaml [hunk 2]
```diff
@@ -2,6 +2,10 @@
 dataset: strogatz_glider2
 description: |
     This is one state of a 2-state dynamic model for a glider. 
+
+    $\dot{v} = - 0.05 \cdot  v^2 - sin (\theta)$
```
Comment (trangdata) [inline datasets/strogatz_glider2/metadata.yaml:6]:
```suggestion
    $\dot{v} = - 0.05 \cdot  v^2 - \sin (\theta)$
```

File: datasets/strogatz_glider1/metadata.yaml [hunk 3]
```diff
@@ -2,6 +2,10 @@
 dataset: strogatz_glider1
 description: |
     This is one state of a 2-state dynamic model for a glider. 
+    
+    $\dot{v} = - 0.05 \cdot  v^2 - sin (\theta)$
```
Comment (trangdata) [inline datasets/strogatz_glider1/metadata.yaml]:
```suggestion
    $\dot{v} = - 0.05 \cdot  v^2 - \sin (\theta)$
```

File: datasets/strogatz_shearflow2/metadata.yaml [hunk 4]
```diff
@@ -2,6 +2,10 @@
 dataset: strogatz_shearflow2
 description: |
     This is one state of a 2-state dynamic model for shear flow. 
+    
+    $\dot{\theta} = \cot (\phi) \cdot cos(\theta)$
+    $ \dot{\phi} = \left(\cos ^2 (\phi) + 0.1 \cdot  \sin^2 (\phi)\right) \cdot sin(\theta)$
```
Comment (trangdata) [inline datasets/strogatz_shearflow2/metadata.yaml]:
```suggestion
    $\dot{\theta} = \cot (\phi) \cdot \cos(\theta)$
    $ \dot{\phi} = \left(\cos ^2 (\phi) + 0.1 \cdot  \sin^2 (\phi)\right) \cdot \sin(\theta)$
```

**Review by trangdata — APPROVED** [review]
Looks great! I added a few backslashes.

File: datasets/strogatz_glider2/metadata.yaml (hunk 2 above)
Comment (lacava) [reply datasets/strogatz_glider2/metadata.yaml:6]: oops, I forgot to commit this before merging. added in 90baba4

**Pull request merged**